oeqPrimaryWS/src/main/java/com/tle/blackboard/webservice/impl/SynchroniseContentThread.java

```diff
@@ -0,0 +1,188 @@
+package com.tle.blackboard.webservice.impl;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.apereo.openequella.integration.blackboard.common.BbContext;
+import org.apereo.openequella.integration.blackboard.common.BbLogger;
+import org.apereo.openequella.integration.blackboard.common.BbUtil;
+import org.apereo.openequella.integration.blackboard.common.content.ContentUtil;
+import org.apereo.openequella.integration.blackboard.common.content.ContentUtil.ContentRegisteredResponse;
+import org.apereo.openequella.integration.blackboard.common.content.ItemInfo;
+import org.apereo.openequella.integration.blackboard.common.content.ItemKey;
+import org.apereo.openequella.integration.blackboard.common.content.RegistrationUtil;
+
+import blackboard.data.content.Content;
+import blackboard.persist.BbPersistenceManager;
+import blackboard.persist.Id;
+import blackboard.persist.content.ContentDbLoader;
+
+/**
+ * @author Aaron
+ */
+public class SynchroniseContentThread extends Thread {
```

Comment:
```suggestion
public class SynchronizeContentThread extends Thread {
```

Reply:
Synchronise is a perfectly acceptable Queen's English :) 
https://www.oxfordlearnersdictionaries.com/definition/english/synchronize  "(British English also synchronise)"

Reply:
It is, but Java standardized on `synchronize`
https://docs.oracle.com/javase/tutorial/essential/concurrency/syncmeth.html

E.G.
https://github.com/equella/equella-blackboard-integration/blob/4f3007a1fc85559c882794d2a4c9dbdf10ab4df2/oeqCommon/src/main/java/org/apereo/openequella/integration/blackboard/common/BbContext.java#L25

Reply:
Thanks for the note on the naming @ChristianMurphy - due to the nature of this need, and the expected lifespan of the B2/WS, this isn't a blocker at this time.